src/styles/main.scss

```diff
@@ -1,3 +1,63 @@
+@import "./mixins.scss";
+@import "./functions.scss";
+
+$cell-size: 100px;
+$border-color: #000;
+$background-color: #eee;
+$font-family: arial;
+$font-size: 30px;
+$padding: 10px;
+$gap: 1px;
+$body-width: 800px;
+$days: mon, tue, wed, thu, fri, sat, sun;
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  min-height: 100vh;
+  display: flex;
+  align-items: center;
+}
+
+.calendar {
+  display: flex;
+  margin: 0 auto;
```

Comment:
You are already centering the calendar vertically with body flex. Let's get rid of this margin and center it with body (justify-content)